modules/apps/object/object-admin-rest-impl/rest-openapi.yaml

```diff
@@ -128,6 +128,8 @@ components:
                 parameterRequired:
                     readOnly: true
                     type: boolean
+                persistedRESTContextPath:
```

Comment:
I don't like this name. The user does not have to know anything about "persisted" word. For them, the field is exposing the rest context path